### PR TITLE
use max volume of -6db for audio artifacts

### PR DESCRIPTION
**Pull request by mosheman5**

### CODE REVIEW

File: soundbay/utils/logging.py [hunk 1]
```diff
@@ -152,8 +152,7 @@ def upload_artifacts(self, audio: torch.Tensor, label: torch.Tensor, raw_wav: to
         # Original wavs batch
 
         artifact_wav = torch.squeeze(raw_wav).detach().cpu().numpy()
-        artifact_wav = artifact_wav / np.max(np.abs(artifact_wav)) * 0.5 #avoid clipping
-        artifact_wav = artifact_wav * 10 ** (volume / (20 * np.log10(1/ np.std(artifact_wav)))) # add dB 
+        artifact_wav = artifact_wav / np.expand_dims(np.abs(artifact_wav).max(axis=1), 1) * 0.5  # gain -6dB
```
Comment (Z30G0D) [inline soundbay/utils/logging.py:155]: Can `artifact_wav == 0`  in some edge case?

Comment (mosheman5) [reply soundbay/utils/logging.py:155]: Yes, in that case we'll just log silent audio.
It won't break anything

Comment (mosheman5) [reply soundbay/utils/logging.py]: Great catch, thanks!
Added a small number so we won't divide by zero

**Review by Z30G0D — APPROVED** [review]
lgtm

**Pull request merged**